src/shared/wizards/multiStepWizard.ts

```diff
@@ -45,6 +47,10 @@ export const WIZARD_GOBACK: Transition = {
     nextState: WizardNextState.GO_BACK,
 }
 
+export const WIZARD_GOBACK_2_STEPS: Transition = {
+    nextState: WizardNextState.GOBACK_2_STEPS
```

Comment:
> Consider using the {@link WIZARD_GOBACK_2_STEPS}

Why? What is the point of these aliases?

Reply:
N/A Implementation has changed.